manim_slides/present.py

```diff
@@ -562,7 +562,15 @@ def __init__(
         self.thread.start()
 
     def keyPressEvent(self, event):
+        self.config = Config()
+
         key = event.key()
+        if self.hide_mouse == False and self.config.HIDE_MOUSE.match(key):
+            self.setCursor(Qt.BlankCursor)
+            self.hide_mouse = True
+        elif self.hide_mouse == True and self.config.HIDE_MOUSE.match(key):
+            self.setCursor(Qt.ArrowCursor)
+            self.hide_mouse = False
         # We send key to be handled by video display
         self.send_key_signal.emit(key)
         event.accept()
```

Comment:
```suggestion
        if self.config.HIDE_MOUSE.match(key):
            if self.hide_mouse:
                self.setCursor(Qt.ArrowCursor)
                self.hide_mouse = False
            else:
                self.setCursor(Qt.BlankCursor)
                self.hide_mouse = True
        else:
            # We send key to be handled by video display
            self.send_key_signal.emit(key)
        event.accept()
```
There is no need to send the key to the thread if we already handled it ;-)